dolphin-dnn/src/main/proto/neural_network.proto

```diff
@@ -23,6 +23,14 @@ message FullyConnectedLayerConfiguration {
   required uint32 num_output = 4;
 }
 
+message PoolingLayerConfiguration {
+  optional string pooling_type = 1;
```

Comment:
We can set a default value for optional parameters as following.
`optional string pooling_type = 1 [default = "max"];`
FYI, here's the Caffe's protocol buffer definition file [[link]](https://github.com/BVLC/caffe/blob/master/src/caffe/proto/caffe.proto).